boards/riscv/tlsr9518adk80d/tlsr9518adk80d_defconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_RISCV_TELINK_B91=y
+CONFIG_SOC_RISCV_TELINK_B91=y
+CONFIG_BOARD_TLSR9518ADK80D=y
+CONFIG_PLIC=y
+CONFIG_RISCV_MACHINE_TIMER=y
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_PINMUX=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_SERIAL=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_CONSOLE_ON_DEV_NAME="uart_0"
```

Comment:
This shouldn't be needed here.

Reply:
I have just checked it, and the solution is not working whiteout "SERIAL", "CONSOLE" and "UART_CONSOLE" configs. 
Other vendors also are enabling these configs in "defconfig" file.

Only UART_CONSOLE_ON_DEV_NAME is optional. I added this config to easy switch between "uart_0" and "uart_1", for debug reasons.

Reply:
I meant `UART_CONSOLE_ON_DEV_NAME`  We normally leave this to come from devicetree and setting of `zephyr,console`